networking/src/Node.hs

```diff
@@ -307,10 +308,9 @@ node mkEndPoint mkReceiveDelay mkConnectDelay prng packing peerData nodeEnv k =
               converse = nodeConverse llnode packing
               unexceptional :: m t
               unexceptional = do
-                  t <- act converse
-                  logNormalShutdown
-                  (LL.stopNode llnode `catch` logNodeException)
-                  return t
+                  (act converse <* logNormalShutdown)
+                  `finally`
+                  (LL.stopNode llnode)
```

Comment:
Is it correct? AFAIU, when an async exception is thrown, `LL.stopNode llnode` will be executed first (due to `finally`) and then `LL.killNode llnode` will be executed as well (due to `onException`). But I suppose we only need to `killNode`.
Also I think it would be ok to leave `logNodeException` here, but I'll leave this decision to you.
And also brackets on this line are redundant.

Reply:
Right my bad, it should be `act converse <* (logNormalShutdown >> LL.stopNode llnode)`